components/dashboard/src/App.tsx

```diff
@@ -143,11 +142,9 @@ function App() {
     }
 
     return (
-        <BrowserRouter>
```

Comment:
Why did we remove BrowserRouter and add added Router  on top of App ?

Reply:
Actually Router is just an alias for BrowserRouter. Anyways I will change it back.